<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recent_kl_widget</class>
 <widget class="QWidget" name="recent_kl_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>km启动窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QListWidget" name="listwidget">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
   border-top: 3px solid #000000; /*黑色上边框*/
}
QListWidget::item {
   background-color: #f0f0f0;  /* 设置项目的背景颜色为浅灰色 */
   border: 1px solid #d3d3d3;  /* 设置项目的边框为1像素的浅灰色实线 */
   padding: 1px;  /* 设置项目内容的内边距为1像素 */
   margin: 1px;  /* 设置项目的外边距为1像素 */
}
QListWidget::item:selected {
   background-color: #a0c4ff;  /* 设置选中项目的背景颜色为浅蓝色 */
   color: white;  /* 设置选中项目的字体颜色为白色 */
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="hint_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sort_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>排序</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>搜索知识库：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="search_pushButton">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
